nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow_schema.json

```diff
@@ -5,195 +5,237 @@
+                    "hidden": true,
+                    "fa_icon": "fas fa-users-cog",
+                    "help_text": ""
+                },
+                "config_profile_url": {
+                    "type": "string",
+                    "description": "Institutional config URL link.",
+                    "hidden": true,
+                    "fa_icon": "fas fa-users-cog",
+                    "help_text": ""
+                }
+            },
+            "fa_icon": "fas fa-university"
         },
-        "Maximum job request limits": {
+        "Max job request options": {
             "type": "object",
-            "description": "Limit the maximum computational requirements that a single job can request",
-            "default": "",
             "properties": {
                 "max_cpus": {
                     "type": "integer",
-                    "description": "Maximum number of CPUs that can be requested  for any single job",
+                    "description": "Maximum number of CPUs that can be requested  for any single job.",
                     "default": 16,
                     "fa_icon": "fas fa-microchip",
-                    "hidden": true
+                    "hidden": true,
+                    "help_text": "Use to set an upper-limit for the CPU requirement for each process. Should be an integer e.g. `--max_cpus 1`"
                 },
                 "max_memory": {
                     "type": "string",
-                    "description": "Maximum amount of memory that can be requested for any single job",
-                    "default": "128 GB",
+                    "description": "Maximum amount of memory that can be requested for any single job.",
+                    "default": "128.GB",
                     "fa_icon": "fas fa-memory",
-                    "hidden": true
+                    "hidden": true,
+                    "help_text": "Use to set an upper-limit for the memory requirement for each process. Should be a string in the format integer-unit e.g. `--max_memory '8.GB'`"
                 },
                 "max_time": {
                     "type": "string",
-                    "description": "Maximum amount of time that can be requested for any single job",
-                    "default": "10d",
+                    "description": "Maximum amount of time that can be requested for any single job.",
+                    "default": "240.h",
                     "fa_icon": "far fa-clock",
-                    "hidden": true
+                    "hidden": true,
+                    "help_text": "Use to set an upper-limit for the time requirement for each process. Should be a string in the format integer-unit e.g. `--max_time '2.h'`"
                 }
-            }
+            },
+            "fa_icon": "fab fa-acquisitions-incorporated"
         },
-        "Institutional config params": {
+        "Pipeline template options": {
```

Comment:
As an end user I'm not sure I'd know what this means..

Reply:
Yeah I agree 🤔 Maybe call this "Generic options"?

Reply:
Sounds good 👍

Reply:
Done https://github.com/nf-core/tools/pull/593/commits/616c1ea52034135906c7fbcef568ffd58b2b1ff8